src/backend/distributed/utils/citus_clauses.c

```diff
@@ -304,6 +310,9 @@ citus_evaluate_expr(Expr *expr, Oid result_type, int32 result_typmod,
 	/* We can use the estate's working context to avoid memory leaks. */
 	MemoryContext oldcontext = MemoryContextSwitchTo(estate->es_query_cxt);
 
+	/* handles default values */
+	FixFunctionArguments((Node *) expr);
```

Comment:
Could this cause a problem in other parts as well? If so should we call this method in `PartiallyEvaluateExpression`?

Reply:
No, we don't call `fix_opfuncids` anywhere else either, as this is the only code path to `ExecEvalExprSwitchContext`